01-Introducing-Molecular-Dynamics/01b-Rigid-Body-Molecular-Dynamics-Simulations.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<p>Some of these equations don't seem to render correctly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
This appears to be a reviewNB issue. They appear correctly in https://github.com/glotzerlab/hoomd-examples/blob/rigid_body_tutorial/01-Introducing-Molecular-Dynamics/01b-Rigid-Body-Molecular-Dynamics-Simulations.ipynb.